src/components/common/RadiationManager.js

```diff
@@ -65,6 +67,24 @@ const Radiations = () => {
     y: height * (1 - SEA.height * (1 + ICE_CAP_HEIGHT)),
   };
 
+  const earthToGasesRadiation = {
+    x: width * 0.75,
+    y: height * 0.6,
+  };
+
+  const gasesToEarthRadiation = {
+    x: width * 0.9,
+  };
+
+  const gasesToSkyRadiation = {
+    x: width * 0.85,
```

Comment:
Are these factors simply for display purposes?

Reply:
Yes, it computes the position where the radiation should begin/stop. Gases don't have a fixed position yet, so that's why it is only based on the height/width of the screen.